### ISSUE TITLE
There are 1041 build warnings

### ISSUE THREAD

**Issue by btipton-ds** [jpbruyere/vkvg#37]
I would say that 99% of these warnings are false alarms, possibly all of them.

But their presence makes it nearly impossible to spot new and dangerous ones as they are added.

I cleaned up about 3,000+ of these at my last employer and uncovered 50 attempts to write out of bounds. The writes only passed because they were writing less than 4 bytes to memory allocated on 4 byte boundaries. It worked but was a ticking bomb.

One I found was 
#include "vk_mem_alloc.h".h

Just a dangling '.h' which the compiler let pass with a warning.

Stuff like this doesn't engender a lot of confidence in the package.

I expect to be using this package as a foundation for some upcoming work and I'm open to doing some cleanup work if requested.

Bob Tipton
Dark Sky Innovative Solutions.

**Comment by jpbruyere** [comment]
I have developed vkvg on linux where no warning remain. I'll try to clean windows ones, I've already cleaned most of them already, I keep you inform.

**Comment by jpbruyere** [comment]
Working with gcc, I targeted c11. But to target msvc, I should limit to C99, I'll try to have it done this week.
What kind of project vkvg would be used for?

**Comment by btipton-ds** [comment]
My career has been in CAD/CAE/Design.

I was lead developer for a package called Freeform which is used to design everything from coins and medical implants to barbie dolls and chocolates.

The huge barrier in this business is cross platform support and the users really don't care what the UI looks like so long as it works.

I've got a first stage pipeline builder running with Vulkan. I've got it rendering buttons on screen using Freetype. My hope was to use SVG for button images.

The immediate target application is hexahedral mesh processing for fluid dynamics work.
